### DISCUSSION TITLE
Piracy Threat Nonsense

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [endless-sky/endless-sky#9320] [Balance, Mechanics, and Interactions]
So to preface, I am aware of the PRs open to change this a bit, and I do hope they make the system less opaque as well as a bit less anti-player

but to focus a bit more on why I call the existing mechanic Nonsense, observe the following

![image](https://github.com/endless-sky/endless-sky/assets/1534478/11415910-f31d-4ee3-a012-c4dc7a995aca)
My cargo is listed as "Bountiful", thus I would assume my cargo holds must contain some large sum of items to be stolen, theoretically

and yet, all eight of my quite heavily armed ships are....empty! 0 cargo at all. This leads to the frustrating situation where every single system visited is swimming in pirates, simply because I am present with my escorts.

I do have to ask why it was ever set up like this, to seemingly calculate based on possible cargo capacity, rather than actual cargo value? Unless it's lumping in outfit value, despite pirates not stealing outfits (and generally just killing you/your escorts?), which would be even more frustrating.

**Comment by mOctave** [comment]
How it's supposed to be set up is as as follows:
The pirates can't _know_ what you have in your cargo.
However, they can see roughly what weapons you have.
Therefore, any risk/reward calculations are going to be based on what they can see, namely your weapons and cargo space, and may not be accurate.

Of course, this logic is flawed, hence the PRs.

**Reply by VaelophisNyx** [reply]
I would like to counter that the Cargo Scanner exists, and that every pirate in the galaxy would probably have at least one stuffed somewhere in their ship(s)

**Reply by mOctave** [reply]
Yes. Which is why this is being changed.
